Remove unneeded apt_source dependency

teemukoo opened the pull request:
Dotdeb is not updated anymore and does not work with Debian 10.

This is a bit hacky solution that should be solved somewhere else.
Perhaps, the apt_source_definitions structure could include the major
version of the distribution.

Commit:
Avoid using dotdeb for Debian 10

This is a bit hacky solution that should be solved somewhere else.
Perhaps, the apt_source_definitions structure could include the major
version of the distribution.

ajsalminen commented:
Was the apt_source config in the role still needed somewhere? Was thinking if it's only for dotdeb and stretch/buster don't use it we could possibly just take it out as unneeded currently.

teemukoo commented:
That's a good point. For Debian, it is indeed only used for dotdeb, so if Jessie compatibility is not needed, the Debian apt_source config could be removed.

However, there is a config for Ubuntu hosts but I don't know if it's used. https://github.com/libraries-fi/ansible-role-nginx/blob/0c40061543f1fa610a5bbe8445a4978cfcda232d/defaults/main.yml#L19-L23

ajsalminen commented:
Yeah, we can do away with the Ubuntu config too. We don't have anything actively using it.

Commit:
Remove unneeded apt_source dependency

The apt_source dependency was used until Debian 8 (Jessie) for dotdeb
repository. However, dotdeb is not updated anymore and it does not
support Debian 9 (Stretch) or later. Therefore, dotdeb is not needed
anymore.

The support for Jessie has ceased in general and this role does not need
to be compatible with it.

The Ubuntu config is not actively used either and can be removed.

Overall, apt_source is not needed anymore in the role.
